openlabcmd/openlabcmd/plugins/base.py

```diff
@@ -26,16 +26,8 @@ def __init__(cls, name, bases, attrs):
     def register_plugin(cls, plugin):
         """Add the plugin to the plugin list and perform any registration logic"""
 
-        # create a plugin instance and store it
-        # optionally you could just store the plugin class and lazily instantiate
-        instance = plugin()
-
         # save the plugin reference
-        cls.plugins.append(instance)
-
-        # apply plugin logic - in this case connect the plugin to blinker signals
-        # this must be defined in the derived class
-        instance.register_signals()
+        cls.plugins.append(plugin)
```

Comment:
-1, So, now the plugin is the plugin class, I guess we could change ``Plugin`` to ``Plugins`` to avoid some confusion? and we can also add some note on it.